config-templates/ibm/ibm-system-storage-block-csi-driver/1.11.1/deployment.yaml

```diff
@@ -10,20 +10,20 @@ items:
   - apiVersion: v1
     kind: Namespace
     metadata:
-      name: "{{ namespace }}"
+      name: "{{namespace}}"
```

Comment:
can you please revert back these changes for `extra space` for all variables enclosed inside double `{{`?

Reply:
done